src/style.css

```diff
@@ -1,3 +1,79 @@
-body {
+* {
```

Comment:
Using the universal selector `*` is not recommended as it impacts performance. Consider applying styles to specific elements or classes instead.